src/egon/data/datasets/electricity_demand_etrago.py

```diff
@@ -0,0 +1,201 @@
+"""The central module containing code to merge data on electricity demand
+and feed this data into the corresponding etraGo tables.
+
+"""
+
+
+import egon.data.config
+import pandas as pd
+from egon.data import db
+from egon.data.datasets import Dataset
+from sqlalchemy.ext.declarative import declarative_base
+
+
+Base = declarative_base()
```

Comment:
`declarative_base` and ` Base ` are not needed in the script later on. So lines 11 and 14 could be deleted.